common/perf-test-core/src/main/java/com/azure/perf/test/core/PerfStressProgram.java

```diff
@@ -209,6 +202,52 @@ public static void run(Class<?> testClass, PerfStressOptions options) {
         }
     }
 
+    private static void printOptions(PerfStressOptions options) {
```

Comment:
nit: would be nice to have a comment before this method that documents the expected string format being parsed

Reply:
Will add this in a future change so I don't need to run core - ci again